### PR TITLE
ROX-21124: Fix upload-dumps-for-downstream GHA job

### PR DESCRIPTION
**Pull request by BradLugo**

### CODE REVIEW

**Comment by BradLugo** [comment]
Let me know if there's a good way to test this, but it looks like this job is only expected to run on master. I'm not sure what the consequences are for overriding that behavior.

File: scripts/ci/lib.sh [hunk 1]
```diff
@@ -282,6 +282,13 @@ get_base_ref() {
         else
             die "Expect PULL_BASE_REF or CLONEREFS_OPTIONS"
         fi
+    elif is_GITHUB_ACTIONS; then
+        # GITHUB_BASE_REF is only set for PRs. Use GITHUB_REF_NAME for Prow's PULL_BASE_REF equivalent.
```
Comment (RTann) [inline scripts/ci/lib.sh:286]: prow?

Comment (RTann) [reply scripts/ci/lib.sh:286]: oh never mind i misread

**Review by RTann — APPROVED** [review]
🚢

**Pull request merged**